udata/core/reuse/forms.py

```diff
@@ -1,8 +1,9 @@
 from udata.forms import ModelForm, fields, validators
 from udata.i18n import lazy_gettext as _
-from udata.models import Reuse, REUSE_TYPES, REUSE_TOPICS
+from udata.models import Reuse
+from udata.core.reuse.constants import REUSE_TYPES, REUSE_TOPICS
 
-from .models import IMAGE_SIZES, TITLE_SIZE_LIMIT, DESCRIPTION_SIZE_LIMIT
+from .constants import IMAGE_SIZES, TITLE_SIZE_LIMIT, DESCRIPTION_SIZE_LIMIT
```

Comment:
Shouldn't we import all from `.constants`?

Reply:
We could, I'm not sure what's best here…